src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -1907,4 +1913,16 @@ namespace UiaOperationAbstraction
     };
 
     bool ShouldUseRemoteApi();
+
+    // Each of these helpers is an overload which populates the cache(s) for

```

Comment:
**Suggestion:** You could make it explicit and differentiate by adding appropriate suffixes:
1. `PopulateCacheForElement`,
2. `PopulateCacheForElementArray` or `PopulateCacheForArrayElements`

Reply:
So, the reason why I have the overload is that the functions that call the helpers are generated by the code gen. Using overloads seemed cleaner than switching over possible types in the code generator.

Reply:
Got'cha. Fine by me. :)

---
In reply to: [459177495](https://github.com/microsoft/Microsoft-UI-UIAutomation/pull/10#discussion_r459177495) [](ancestors = 459177495)